src/ImageProcessorCore/Formats/Jpg/JpegDecoderCore.cs

```diff
@@ -3,6 +3,8 @@
 // Licensed under the Apache License, Version 2.0.
 // </copyright>
 
+using ImageProcessorCore.Formats.Tiff;
```

Comment:
I've deliberately kept the namespace flat for the image formats to make it easier for the end user so use `ImageProcessorCore.Formats`. 

Unfortunately .xproj types always default to the folder and you can't ignore folder namespaces in those project types with Resharper due to a lack of support in VS for now you have to manage them manually.